container.go

```diff
@@ -117,12 +118,23 @@ func showContainerCheckpoint(checkpointDirectory, input string) error {
 		// Get dump statistics with crit
 		dumpStats, err := crit.GetDumpStats(cpDir.Name())
 		if err != nil {
-			return fmt.Errorf("unable to display checkpointing statistics: %w", err)
+			return fmt.Errorf("failed to get dump statistics: %w", err)
 		}
 
 		renderDumpStats(dumpStats)
 	}
 
+	if psTree {
+		c := crit.New("", "", checkpointDirectory, false, false)
```

Comment:
> Error: failed to get process tree: Error opening binary file: open /tmp/checkpointctl1506966507/pstree.img: no such file or directory

Reply:
Ack, figured this out. The temporary directory being created is one level above the checkpoint directory. I need to append that path here.

Reply:
I tried listing the files present in the temporary directory, and got this:
```
/tmp/checkpointctl65800022/config.dump
/tmp/checkpointctl65800022/spec.dump
/tmp/checkpointctl65800022/stats-dump
```